### PR TITLE
[Pibbble][##109714428] Enable views ranked by likes and number of views

### PR DESCRIPTION
**Pull request by sirolad**

### CODE REVIEW

**Comment by unicodeveloper** [comment]
Your commit messages does not follow the commit standard @andela-sakande . Please rewrite them to follow the standards for this project.

File: app/Http/Controllers/PagesController.php [hunk 1]
```diff
@@ -63,4 +63,33 @@ public function help()
...
+
+        return view('pages.comments', ['projects' => $projects]);
+    }
+
+    /**
+     * @return views.blade.php
+     */
+    public function views()
+    {
+       $projects = Project::orderBy('views', 'desc')->paginate(12);
```
Comment (unicodeveloper) [inline app/Http/Controllers/PagesController.php:78]: Please fix this

File: app/Http/Controllers/PagesController.php [hunk 2]
```diff
@@ -63,4 +63,33 @@ public function help()
...
+
+        return view('pages.views', ['projects' => $projects]);
+    }
+    /**
```
Comment (unicodeveloper) [inline app/Http/Controllers/PagesController.php:82]: There should be a newline between line 85 and 86

File: app/Http/Controllers/PagesController.php [hunk 3]
```diff
@@ -63,4 +63,33 @@ public function help()
...
+     * @return likes.blade.php
+     */
+    public function likes()
+    {
+        $projects = Project::orderBy('likes', 'desc')->paginate(12);
+
+        return view('pages.likes', ['projects' => $projects]);
```
Comment (unicodeveloper) [inline app/Http/Controllers/PagesController.php:89]: Why do we have different views for `likes` and `views` ?

Comment (sirolad) [inline app/Http/Controllers/PagesController.php:89]: Because each of the pages reflect either most viewed or most liked. They are different criteria.

Comment (unicodeveloper) [inline app/Http/Controllers/PagesController.php:89]: You don't need to have different views, use a query string to determine the kind of content that shows on the landing page. 

For instance, `pibbble-staging.herokuapp.com/submissions?sort=views` for views and `pibbble-staging.herokuapp.com/submissions?sort=likes` and `pibbble-staging.herokuapp.com/submissions?sort=comments` is a very good way of doing it.

So when the user clicks on this links, it performs those queries based on these query strings and displays them on the normal page, thus eliminating the need for a different page for `likes` and `views`

**Pull request closed without merging**